api/lib/database/methods.js

```diff
@@ -22,7 +22,7 @@ const getAccountsNotifications = ({
     ],
     `where: {
       networkId: {_eq: "${networkId}"},
-      resourceId: {_in: [${addresses.map((address) => `"${address}"`)}]},
+      resourceId: {_in: ["${address}"]},
```

Comment:
```suggestion
      resourceId: {_eq: "${address}"},
```

Reply:
:see_no_evil: 

Then we are back to the start line! We already tried this first, do you remember? It is much slower

Reply:
No, it is not. The problem was, that you were loading all validators in parallel now you are loading only one. Actually the code does the same it is just clearer.

Reply:
OK, I will try it, but I also recall querying one single validator being quite slow with `_eq`

Reply:
OK, I just tried both and both are very slow :sweat_smile: Maybe I am just having bad connection

Reply:
With `_eq: "${address}"`: ~21s

With `_in: ["${address}"]`: ~4s

You can try yourself

Reply:
You mean the cache? I will try now the opposite order & see

Reply:
OK, the results now were:

With `_eq: "${address}"`: ~11s

With `_in: ["${address}"]`: 3.48s